### DISCUSSION TITLE
Custom Fields for Media

### DISCUSSION THREAD

**Discussion by olivdee** [joomla/joomla-cms#44535] [2. Cold Features]
There are custom fields in "Users", there are also custom fields in "Content" - but only for articles and categories, why not as well for "media"? For example (depending on project needs), media could have "caption" and "alt tag" and "title" and "link" etc. right inside them from the media manager (if needed), so one would not need to repeat those information for example inside article intor/article images - or somewhere else needed... I understand, that in 90% of all cases, this is not needed, but in my opinion, it would be a logical addition to custom fields for articles and categories, especially because both are part of "content" and only "media" has been missed out...

Any plans for such an enhancement in future?

**Comment by brianteeman** [comment]
There are no plans for this as that would require media becoming a database based system and not a file based system. Also it is not generally a good idea as any alt text description should be generated in the context of the content where it is being used. Read more about that https://magazine.joomla.org/all-issues/june-2024/ai-generated-alt-text

**Reply by olivdee** [reply]
Thanks for the quick reply! Makes sense what you are writing. The fields inside "article" > "images & links" are doing their job. We are using those currently for a galerie feed, with simple images, or videos (if "Link A") is filled out. Any quick idea how to stop all those articles containing only an image and feeding the galerie appearing in the frontend - is there an easy way in Joomla to route an entire categories content (all articles) to somewhere else (maybe the category parent menu - category blog, or other page from menu)?

**Reply by brianteeman** [reply]
Not sure what you are trying to achieve.
